pages/index.js

```diff
@@ -2,31 +2,24 @@ import React from 'react';
 import { connect } from 'react-redux';
 import Home from './../src/components/home/Home';
 import { Actions } from './../src/actions';
-import { getLimitPosts } from './../src/webAPI'
+import { withRedux } from './../src/withRedux';
 
 const HomeContainer = props => {
  return (<Home {...props} />);
 }
 
 const mapStateToProps = ({ posts }, props) => ({
- postList: posts.postList.length > 0 ? posts.postList : props.ssrPostList,
+ postList: posts.postList || [],
```

Comment:
add error handle in case postList is not an array, but it will be better to handle inside the reducer.
Another package we can use to handle this - https://github.com/reduxjs/reselect